traits/ctraits.c

```diff
@@ -4223,7 +4229,7 @@ static int
 _set_trait_comparison_mode(trait_object *trait, PyObject *value, void *closure)
 {
     long comparison_mode = PyLong_AsLong(value);
-    
+
```

Comment:
VS Code did this, not me! (Apparently there was some trailing whitespace in ctraits.c)